its/src/test/java/com/sonar/maven/it/suite/DependencyTest.java

```diff
@@ -43,7 +44,6 @@
   public void deleteData() {
     // Design features have been dropped in 5.2
     assumeTrue(!orchestrator.getServer().version().isGreaterThanOrEquals(5, 2));
-    orchestrator.resetData();
```

Comment:
I think a cleanup of analyzed projects should be done here, like : https://github.com/SonarSource/sonar-dotnet/blob/98c0671a0b71ec36f1fc7a237aa51a7b6e5f57d7/its/src/test/java/com/sonar/it/shared/TestUtils.java#L187